eval.c

```diff
@@ -1079,11 +1079,21 @@ rb_mod_prepend(int argc, VALUE *argv, VALUE module)
 {
     int i;
     ID id_prepend_features, id_prepended;
+    VALUE new_mod;
 
     CONST_ID(id_prepend_features, "prepend_features");
     CONST_ID(id_prepended, "prepended");
 
-    rb_check_arity(argc, 1, UNLIMITED_ARGUMENTS);
+    if(rb_block_given_p()) {
+      new_mod = rb_module_new();
```

Comment:
adjust the style and indent.
in general, not only ruby, you should respect the style in the original files, not your favorite style.